meta-mel/conf/distro/mel.conf

```diff
@@ -320,11 +320,10 @@ DL_LICENSE_INCLUDE ?= "*"
 INHERIT += "archive-release-downloads"
 
 # For our forks, also point an 'upstream' remote at upstream
-UPSTREAM_URL = ""
-UPSTREAM_URL_bitbake = "https://github.com/openembedded/bitbake"
-UPSTREAM_URL_openembedded-core = "https://github.com/openembedded/openembedded-core"
+UPSTREAM_URL_meta-virtualization = "git://git.yoctoproject.org/meta-virtualization"
+FORKED_REPOS ?= "meta-virtualization"
```

Comment:
hmmmmm so in order to do this right the BSP layers should append to this appropriately? We do fork vendor repos sometimes.

Reply:
Yeah, mel.conf has the baseline for flex os, but bsps can add to it when we fork those layers. This only applies to public layers, and it just arranges for git remotes to exist in the installed git repositories so the customer can easily fetch both the fork and upstream, as a usability improvement. That said, if one is missed, it's certainly  non-critical, which is how this got out of date in the first place.